src/components/Layout/NavigationLayout/GenreNavigation/GenreNavigation.tsx

```diff
@@ -32,9 +30,20 @@ const GenreNavigation = (): JSX.Element => {
 
   function updateFilters(newFilters): void {
     query.set(FILTER_BY, newFilters);
-    history.replace({
-      search: query.toString(),
+    router.replace({
```

Comment:
Is this first `.replace()` call really needed? Because it is called 1 more time in one of the conditions

Reply:
You are right it is redundant